Moloch - fix moloch-sessions-json command (#12846)
demisto/content: Packs/Moloch/ReleaseNotes/1_0_1.md

#### Integrations
##### Moloch

> - Fixed the **moloch-sessions-json** command, which used to return faulty results.

Reviewer: ```suggestion
- Fixed an issue where the ***moloch-sessions-json*** command would return faulty results.
```
